### PR TITLE
[RTM] Add template blocks to be_login.php

### PR DESCRIPTION
**Pull request by CMSworker**
Added blocks like here:
https://github.com/contao/core-bundle/blob/master/src/Resources/contao/templates/backend/be_main.html5

With blocks it should be possible to add content in a child template (Template inheritance) like described in the book here: https://docs.contao.org/books/manual/4.0/en/04-managing-content/templates.html

Use case: Wanted to add a custom logo with CSS in the Login page...

### CODE REVIEW

**Comment by leofeyer** [comment]
Thank you @CMSworker.

File: src/Resources/contao/templates/backend/be_login.html5 [hunk 1]
```diff
@@ -2,64 +2,71 @@
...
-    $objCombiner->add('system/themes/'. $this->theme .'/login.css');
-    echo $objCombiner->getCombinedFile();
-  ?>">
-  <?= $this->stylesheets ?>
+    <?php $this->block('meta'); ?>
+      <meta name="generator" content="Contao Open Source CMS">
+      <meta name="viewport" content="width=device-width,initial-scale=1.0,shrink-to-fit=no">
+      <meta name="referrer" content="origin">
+      <meta name="robots" content="noindex,follow">
+    <?php $this->block('meta'); ?>
```
Comment (ausi) [inline src/Resources/contao/templates/backend/be_login.html5]: Should be `$this->endblock()`

File: src/Resources/contao/templates/backend/be_login.html5 [hunk 2]
```diff
@@ -2,64 +2,71 @@
...
-        </div>
-      </form>
-    </main>
-    <?= $this->messages ?>
-  </div>
+        </form>
+      </main>
+      <?= $this->messages ?>
+    </div>
+  <?php $this->block('container'); ?>
```
Comment (ausi) [inline src/Resources/contao/templates/backend/be_login.html5]: Should be `$this->endblock()`

**Comment by leofeyer** [comment]
Fixed in 552e627879f60ff005bf86dba4ae7731b3b64d55. Thank you for your careful review @ausi .

**Pull request merged**